examples/build_report.py

```diff
@@ -55,21 +55,33 @@
 group = Squad().group(args.group)
 project = group.project(args.project)
 build = project.build(args.build)
-
+testruns = Squad().testruns(build=build.id)
 squad_envs = Squad().environments(project=project.id)
 squad_suites = Squad().suites(project=project.id)
-envs = {}
+envs_tr_combo = {}
 for env in squad_envs.values():
     suites = {}
-    envs.update({env.slug: suites})
-    tests = Squad().tests(test_run__environment=env.id, test_run__build=build.id)
-    # group tests by suite
-    for test in tests.values():
-        suite_name = squad_suites[test.suite].slug
-        if suite_name in suites.keys():
-            suites[suite_name].append(test)
-        else:
-            suites.update({suite_name: [test]})
+    metric_suites = {}
+    for testrun in testruns.values():
+        envs_tr_combo.update({(env.slug, testrun): (suites, metric_suites)})
```

Comment:
I don't like this. You're giving suites in random order which looks ugly in the final report. Also using tuple as a key is asking for trouble.